### ISSUE TITLE
Keydown event interception prevents navigation with focus in overview mode

### ISSUE THREAD

**Issue by roman01la** [shower/shower#215]
When navigating with Tab/Shift+Tab keys in overview mode, the focus remains on document. This makes not possible to read-out selected slide.

This is due to implementation of [programmatic navigation](https://github.com/shower/core/blob/aa8a6bfc36d8808a47e9b44f915bda13200426ff/shower.js#L988-L993).

I think a better way would be to allow navigation with focus.

**Comment by pepelsbey** [comment]
This is just just the way how Shower works: in the list mode you’re navigating between slides, in full mode you’re navigating inside of the slide. But this is only true for visual users.

I guess the best way to fix it for screen readers would be to drop support for Tab shortcuts, am I right?

**Comment by roman01la** [comment]
Yes. And also add `tabindex="0"` to every slide/`<section>` element to make it actually focusable.
I think this can be done for full mode as well: just show a slide which is currently in focus.

**Comment by chaals** [comment]
I was looking at this - I am trying to share a slide presentation with a screenreader user.

What you really want to do when navigating, given the way it is built, is bring the focus to the first focusable thing in the slide.

For my use case this would be the slide's heading with a tabindex - Powerpoint passes the title(?) of the current slide to JAWS as it moves between them.

**Comment by chaals** [comment]
Hmm. Just disabling tab navigation does some odd things (at least in Yandex browser and Firefox). Time to check more carefully and see if it is because I have an old version.
